recipes/perl/storage/storage.pm

```diff
@@ -240,5 +240,149 @@ sub post_disk_pool {
     }
 }
 
+# Add a replication target on a disk volume
+sub post_add_replication_target_on_dv {
+    my $arguments_count = scalar(@_);
+    if ($arguments_count != 5) {
+        print "ERROR :: Incorrect number of arguments passed to post_add_replication_target_on_dv()\n";
+        print "Usage : post_add_replication_target_on_dv( <Master Server Hostname>, <Token>, <StorageServerId> <DiskVolumeId> <Payload>) \n";
+        return;
```

Comment:
I think the usual way of bailing out when a module function is misused is to call [`croak`](https://perldoc.perl.org/Carp.html).